package.json

```diff
@@ -28,7 +28,7 @@
     "docs:build": "build-storybook -o docs",
     "docs:serve": "http-server -p 6006 docs",
     "lint:format": "prettier --check .",
-    "lint:css": "stylelint \"./scss/**/*.scss\" \"./styleguide/**/*.scss\"",
```

Comment:
Dropping this as we're about to remove storybook so we don't really care about linting these styles, and most core rules don't apply here anyway.